client/landing/stepper/declarative-flow/internals/hooks/use-launchpad-decider/index.tsx

```diff
@@ -1,10 +1,12 @@
-import { updateLaunchpadSettings } from '@automattic/data-stores';
+import { OnboardSelect } from '@automattic/data-stores';
```

Comment:
Nit: Should this not be a type import?
```suggestion
import type { OnboardSelect } from '@automattic/data-stores';
```